mkdocs.yml

```diff
@@ -56,6 +56,7 @@ pages:
 - For Developers:
     - How to Build: Building.md
     - Instrumentation: Instrumentation.md
+    - Integration tests: Testing.md
```

Comment:
broken link, the file is actually now called `Test.md` though it should probably be something like `Centaur.md`